examples/react-native-todo-list/App.tsx

```diff
@@ -9,19 +9,32 @@ import {
   KeyboardAvoidingView,
   Platform,
 } from "react-native";
-import { useOthers, useList, useUpdateMyPresence } from "./liveblocks.config";
+import {
+  useOthers,
+  useUpdateMyPresence,
+  useStorage,
+  useMutation,
+} from "./liveblocks.config";
 import Header from "./src/components/Header";
 import Todo from "./src/components/Todo";
 import TextInputWithButton from "./src/components/TextInputWithButton";
 import NoMoreTodos from "./src/components/NoMoreTodos";
 
 const App = () => {
-  const todos = useList("todos");
-  const others = useOthers();
+  const todos = useStorage((root) => root.todos);
+  const userCount = useOthers((others) => others.length);
   const updateMyPresence = useUpdateMyPresence();
-  const isSomeoneTyping = others
-    .toArray()
-    .some((user) => user.presence?.isTyping);
+  const isSomeoneTyping = useOthers((others) =>
+    others.some((user) => user.presence?.isTyping)
```

Comment:
Since it's now mandatory to set `initialPresence` in the RoomProvider, we will always know a user's presence value, so this uncertainty is a thing of the past.

```suggestion
    others.some((user) => user.presence.isTyping)
```

Reply:
gotcha! fixed!